game/src/main/scala/io/aigar/game/Grid.scala

```diff
@@ -11,10 +11,20 @@ object Grid {
 class Grid(val width: Int, val height: Int) {
   val random = new Random()
 
-  def randomPosition = {
+  def randomPosition: Vector2 = {
     val x = random.nextFloat() * width
     val y = random.nextFloat() * height
-    new Vector2(x, y)
+    Vector2(x, y)
+  }
+
+  def randomRadiusPosition: Vector2 = {
```

Comment:
Could you link [the SO answer](http://stackoverflow.com/a/5838055/395386) here please?